### DISCUSSION TITLE
pkg update fail

### DISCUSSION THREAD

**Discussion by yazigegeda** [termux/termux-app#3069] [General]
![Screenshot_2022-10-31-12-19-54-40_607bdd22fe69f77fcb7d6df7648bb058](https://user-images.githubusercontent.com/63853093/198931214-5572804a-7815-4c84-9970-4303e058e9d7.jpg)

How can I solve this problem

**Comment by 2096779623** [comment]
> Remember that VMOS and F1VM environments are NOT SUPPORTED!
These "virtual" environments are not supported by Termux. They have a number of issues including broken apt-key.

https://github.com/termux/termux-packages/issues/6726
